client/v3/concurrency/session.go

```diff
@@ -108,14 +110,16 @@ type sessionOptions struct {
 }
 
 // SessionOption configures Session.
-type SessionOption func(*sessionOptions)
+type SessionOption func(*sessionOptions, *zap.Logger)
 
 // WithTTL configures the session's TTL in seconds.
 // If TTL is <= 0, the default 60 seconds TTL will be used.
 func WithTTL(ttl int) SessionOption {
-	return func(so *sessionOptions) {
+	return func(so *sessionOptions, lg *zap.Logger) {
 		if ttl > 0 {
 			so.ttl = ttl
+		} else {
+			lg.Info("TTL should be > 0, using default", zap.Int64("default-session-ttl", defaultSessionTTL))
```

Comment:
'using default' might be misleading. 

See what happens in case of akward setup: 

 `..., WithTTL(30), WithTTL(0),`
 
 We would land with 30, but the message would say: `using default`. 

How about: 
  lg.Info("WithTTL(): TTL should be > 0, preserving current ttl", zap.Int64("current-ttl", so.ttl)) ?